app/src/main/java/org/simple/clinic/bloodsugar/unitselection/BloodSugarUnitSelectionDialog.kt

```diff
@@ -10,7 +10,7 @@ import androidx.appcompat.app.AppCompatDialogFragment
 import androidx.fragment.app.FragmentManager
 import org.simple.clinic.R
 
-class BloodSugarUnitSelectionDialog : AppCompatDialogFragment() {
+class BloodSugarUnitSelectionDialog : AppCompatDialogFragment(), BloodSugarUnitSelectionUiActions {
```

Comment:
This could actually be a separate commit which just implements the UI actions on the dialog since this change is not necessary for handling the event in the `update()` function.